deployment-repository/src/main/java/org/jboss/as/repository/ContentRepository.java

```diff
@@ -53,7 +58,7 @@
  * @author John Bailey
  */
 public interface ContentRepository {
-
+    String UNSUPPORTED_PROPERTY = "org.wildfly.unsupported.content.repository.obscolescence";
```

Comment:
BTW my vague concern here is code like wildfly-swarm or torquebox treating that string as some sort of dependable API. Just putting @Deprecated on it may suffice.